docs/en-US/component/tag.md

```diff
@@ -105,10 +105,11 @@ tag/checkable
 
 ### CheckTag Attributes
 
-| Name                      | Description      | Type                                                               | Default |
-| ------------------------- | ---------------- | ------------------------------------------------------------------ | ------- |
-| checked / v-model:checked | is checked       | ^[boolean]                                                         | false   |
-| type ^(2.5.4)             | type of CheckTag | ^[enum]`'primary' \| 'success' \| 'info' \| 'warning' \| 'danger'` | primary |
+| Name                      | Description                       | Type                                                               | Default |
+|---------------------------|-----------------------------------| ------------------------------------------------------------------ | ------- |
+| checked / v-model:checked | is checked                        | ^[boolean]                                                         | false   |
+| disabled    ^(2.8.1)              | whether the check-tag is disabled | ^[boolean]                                                         | false   |
```

Comment:
Now the table format may look messy. You can run `pnpm format` to adjust it.

Reply:
okay, I have completed it.